content/faq.md

```diff
@@ -79,12 +79,64 @@ In most cases, that is a statement on a provider's website or the result page of
 The source of a property can also be the property itself.
 That is the case when the property is a web page (e.g., a web registration page).
 
+Furthermore, several properties are updated automatically on a daily basis.
+E.g., our web bot retrieves ratings while our XMPP bot retrieves registration support and upload limits directly from the providers' XMPP server.
+
 All sources are stored in the {{< external-link text="providers file" url="https://invent.kde.org/melvo/xmpp-providers/-/blob/master/providers.json" >}}.
 That file is the data source of all provider properties.
 This website's provider listings and details are based on it.
 
 {{< spacer size="middle" >}}
 
+## How can server operators provide properties via XMPP?
+
+Our XMPP bot requests many properties via XMPP from the providers' servers.
+Here are some hints on how to configure the server to provide the properties.
+
+### Account Creation via XMPP Apps
+
+For determining whether a server supports the creation of accounts via XMPP apps, [XEP-0077: In-Band Registration](https://xmpp.org/extensions/xep-0077.html) is used.
+
+Please enable that feature to simplify the onboarding process for new users:
+* ejabberd: [mod_register](https://docs.ejabberd.im/admin/configuration/modules/#mod-register)
+* Prosody: [mod_register_ibr](https://prosody.im/doc/modules/mod_register_ibr)
+
+### Account Creation via Web Page
+
+For retrieving a web page that can be used to create accounts, [XEP-0077: In-Band Registration - Redirection](https://xmpp.org/extensions/xep-0077.html#redirect) is used.
+
+Please use that feature to either provide an alternative way for users to create an account or if your server does not support account creation via XMPP apps:
+* ejabberd: [mod_register](https://docs.ejabberd.im/admin/configuration/modules/#mod-register) - `redirect_url`
+* Prosody (alternative modules):
+    * [mod_register_redirect](https://modules.prosody.im/mod_register_redirect.html) - `registrarion_url` (the typo comes from its documentation)
```

Comment:
```suggestion
    * [mod_register_redirect](https://modules.prosody.im/mod_register_redirect.html) - `registration_url` 
```

From the module's code (the typo is not present in the code):
```lua
local url = module:get_option_string("registration_url", nil)
```

Reply:
Thanks to @mwild1, it is fixed now in the documentation as well.